README.md

```diff
@@ -65,6 +65,38 @@ function setAction(Action $action) {
 }
 ```
 
+## Comparison
```

Comment:
It should come after the main documentation: it's a detail, it shouldn't distract from the existing documentation.